backend/tests/test_xslt.xsl

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would recommend more descriptive filename, for exemple:  xslt_public_to_prod.xslt

Reply:
yes, good idea